source/features/infinite-scroll.tsx

```diff
@@ -29,10 +29,8 @@ const inView = new IntersectionObserver(([{isIntersecting}]) => {
 });
 
 function init(): Deinit {
-	const selectorObserver = observe('.ajax-pagination-btn', {
-		add(button) {
-			inView.observe(button);
-		},
+	observe('.ajax-pagination-btn', button => {
+		inView.observe(button);
 	});
```

Comment:
Actually this can be two `element-ready` calls, but our selector-observer is so lightweight 😆

Reply:
Just realized yet another limitation of animations: they don't start when the tab is hidden.

This is the same rAF limitation described here:

- https://github.com/refined-github/refined-github/issues/4312#issuecomment-831007823

Reply:
You have stated it here: https://github.com/refined-github/refined-github/pull/5886#issuecomment-1204363045

It's fine in this case, we don't need to load a hidden tab.